config/lbx/config.cpp

```diff
@@ -66,15 +82,25 @@ void setup() {
     static InputSource *input_sources[] = { gpio_input };
     size_t input_source_count = sizeof(input_sources) / sizeof(InputSource *);
 
-    // Hold B on plugin for Brook board mode.
+    // Hold start on plugin for Brook board mode.
     pinMode(pinout.mux, OUTPUT);
-    if (button_holds.b) {
+    if (button_holds.start) {
```

Comment:
Why not use B? Is there a conflict with brook board button holds?

Reply:
That makes sense, but my only concern is that it wouldn't match the list of button holds in the readme, so I anticipate LBX users would come to me asking why brook mode isn't working. I think it should stay as B for now, for consistency. Maybe in a future release I'll rethink button holds again. Start is used for entering bootsel mode on Pico at the moment and I want to keep things consistent wherever possible.